book.rb

```diff
@@ -0,0 +1,11 @@
+# this class represents the books
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] There should be a method for adding rentals here, just as you have it in person class but would have parameters for person and date so that it would serve that purpose but peculiar rentals here. Remember that in the method, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association

Reply:
Thank you @Olamarx. Done!